lib/byebug/commands/list.rb

```diff
@@ -113,7 +113,7 @@ def move(line, size, direction = "+")
     # @param max [Integer] Upper bound
     #
     def display_lines(min, max)
-      puts "\n[#{min}, #{max}] in #{frame.file}"
+      puts "\n[#{min}, #{max}] in #{frame.file}:#{(max+min)/2}"
```

Comment:
I think it's simpler to write `frame.line` here?

Reply:
You are right, it is more proper like that. I will modify as follows and do the things in your other comment at the weekend.
```suggestion
      puts "\n[#{min}, #{max}] in #{frame.file}:#{frame.line}"
```